### PR TITLE
Game moment inertia

### PR DESCRIPTION
**Pull request by oluowoj**
Added a new definition into DataDef.hs

files updated:  drasil-example/Drasil/GamePhysics/Unitals.hs
                        drasil-example/Drasil/GamePhysics/DataDefs.hs

new pull request still has some extra unrelated info.

### CODE REVIEW

**Comment by oluowoj** [comment]
@Mornix  as we discussed last week, I created a new branch from a fresh master, but it looks like I still see the same issue and obviously the build failed.

**Comment by JacquesCarette** [comment]
That's because there are differences in the logs -- see line 912 or so in the travis log https://travis-ci.org/JacquesCarette/Drasil/builds/538314079?utm_source=github_status&utm_medium=notification

**Comment by Mornix** [comment]
@oluowoj It looks like your `master` may be out of sync with upstream. If that's the case, you are on `master` you can run `git reset --hard origin/master` to reset your master to upstream master.

**Comment by JacquesCarette** [comment]
The build is still broken - it must not just be a matter of merging. Please look into the logs to find out what the reported problem is.

File: code/stable/gamephys/Website/Chipmunk_SRS.html [hunk 1]
```diff
@@ -1719,6 +1726,63 @@ <h3>Data Definitions</h3>
...
+<td>
+<ul class="hide-list-style-no-indent">
+<li>
+<em><b>I</b></em> is the moment of inertia (kg&sdot;m<sup>2</sup>)
+</li>
+<li>
+<em>m<sub>j</sub></em> is the mass of the j-th particle (kg)
+</li>
+<li>
+<em>r<sub>j</sub></em> is the distance perpendicular distance between the j-th particle and the angle of rotation (m)
```
Comment (smiths) [inline code/stable/gamephys/Website/Chipmunk_SRS.html]: This description isn't quite right. It should say "... the distance between the j-th particle and the axis of rotation (m)".

Comment (oluowoj) [reply code/stable/gamephys/Website/Chipmunk_SRS.html]: Thanks, updated

**Comment by oluowoj** [comment]
@smith , were you thinking to have this?
image
--moment of inertia of a rigid body with respext to a specific axis--

**Comment by oluowoj** [comment]
new pull request #1439 has all changes for the PR, this PR can be closed and branch deleted.

@smiths, please review changes in the new PR

**Comment by smiths** [comment]
@oluowoj your proposed equation is along the right lines, but I don't like the differential over the mass.  I'd rather introduce density and have the integral over the area.

IntegralEq

You can see this equation at:

https://en.wikipedia.org/wiki/Moment_of_inertia

This would be the proper way to present the moment of inertia.  This is a general formulation.  It works for any geometry, mass distribution and coordinate system.  Eventually we want to introduce the knowledge in this form and then customize it for the specific example.  We can hold off on this though, so that we don't get too distracted by the physics.  :smile:

File: code/stable/gamephys/Website/Chipmunk_SRS.html (hunk 1 above)
Comment (oluowoj) [reply code/stable/gamephys/Website/Chipmunk_SRS.html]: This has been updated
image

**Comment by JacquesCarette** [comment]
As requested, closing this one. Conversation should move to #1439.

**Pull request closed without merging**